collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/RequiredQuestionTest.kt

```diff
@@ -44,6 +45,17 @@ class RequiredQuestionTest {
             .swipeToNextQuestionWithConstraintViolation("Custom message")
     }
 
+    @Test
+    fun errorMessageDisappearsAfterActivityRecreation() {
+        rule.startAtMainMenu()
+            .copyForm("required_question_with_custom_error_message.xml")
+            .startBlankForm("required_question_with_custom_error_message")
+            .swipeToNextQuestionWithConstraintViolation("Custom message")
+            .rotateToLandscape(FormEntryPage("required_question_with_custom_error_message"))
+            .rotateToPortrait(FormEntryPage("required_question_with_custom_error_message"))
```

Comment:
I think we could just rotate once here right?

Reply:
If we rotate to landscape the edit text will be displayed in that full-screen mode and there will be no error no matter if the bug is fixed or not.